### PR TITLE
added callout for multi auth for datastore

### PR DESCRIPTION
**Pull request by ykethan**
_Description of changes:_
The Pr adds a callout for customers using datastore with multi auth types. added a link to library docs for configuration.

image

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/pages/console/uibuilder/eventhandling.mdx [hunk 1]
```diff
@@ -91,6 +91,13 @@ This will render the component within an HTML `<form />` element.
 
 </Callout>
 
+<Callout warning>
+
+When working with DataStore, for some use cases you will want to use multiple authorization types. For example, an app might use `API Key` for public content and `Cognito User Pool` for personalized content once the user logs in.
```
Comment (cwoolum) [inline src/pages/console/uibuilder/eventhandling.mdx:96]: We should tell customers that forms uses DataStore under the hood and permissions are based on that.

Maybe `Amplify Forms use DataStore to save and load data. In most cases, you will want to have authentication for your forms. For example, an app might use 'API Key' for public content and 'Cognito User Pool' for personalized content once the user logs in.` or something to that effect.

Comment (chrisbonifacio) [reply src/pages/console/uibuilder/eventhandling.mdx:96]: @cwoolum @ykethan 
Maybe this?

>Amplify Forms use DataStore to save and load data. In most cases, you will want to have authentication for your forms. For example, an app might use 'API Key' for public content and 'Cognito User Pool' for personalized content once the user logs in. However, DataStore will only use the default authentication type and not additional types unless configured with `MULTI_AUTH` enabled. To enable multiple authentication types in DataStore & Amplify Forms, please refer to [Configure Multiple Authorization Types](/lib/datastore/setup-auth-rules/#configure-multiple-authorization-types).

Comment (ykethan) [reply src/pages/console/uibuilder/eventhandling.mdx:96]: hey @chrisbonifacio, i like the suggestion. I will add this now.

**Review by chrisbonifacio — APPROVED** [review]
LGTM!

**Review by cwoolum — APPROVED** [review]

**Review by johnpc — APPROVED** [review]

**Review by wrpeck — APPROVED** [review]

**Pull request merged**